src/k8s/pkg/k8sd/types/cluster_config_kubelet.go

```diff
@@ -46,6 +55,10 @@ func KubeletFromConfigMap(m map[string]string) (Kubelet, error) {
 	if v, ok := m["cluster-domain"]; ok {
 		c.ClusterDomain = &v
 	}
+	if v, ok := m["control-plane-taints"]; ok {
+		s := strings.FieldsFunc(v, func(c rune) bool { return c == ',' })
+		c.ControlPlaneTaints = &s
+	}
```

Comment:
Do we ever need to update this flag in running kubelets? Probably not, and this would also find its way to worker nodes as well.

I'd recommend not exposing this in the configmap at all.